app/actions/mark_pr_in_progress.rb

```diff
@@ -7,10 +7,15 @@ def initialize(parser, pull_request)
   end
 
   def self.matches(parser, _pull_request)
-    parser.event_type == "pull_request_review_comment" &&
+    comments?(parser.event_type) &&
       EXCLUDED_USERS.exclude?(parser.comment_user_login)
   end
 
+  def self.comments?(event_type)
+    ["pull_request_review_comment", "issue_comment"].include?(event_type)
```

Comment:
I like this extraction! What do you think about pulling strings out into a constant?

```ruby

TYPES_WITH_COMMENTS= ["pull_request_review_comment", "issue_comment"]
```

Alternatively, could we look for `_comment` in the string without needing to specify the entire string?

Reply:
> Alternatively, could we look for _comment in the string without needing to specify the entire string?

I like this idea!
Can I make assumption? it is most likely a `comment` object when it is ended by `_comment`. So `_comment` in the middle won't be matched since it might not be a `comment` object

so far, there are 3 Github's webhook contains `_comment`: 
- `commit_comment` -> it gonna match with the regex but will get `NullPullRequest.new`, so I think I can ignore it
- `pull_request_review_comment `
- `issue_comment`

https://developer.github.com/v3/activity/events/types/